scripts/build.js

```diff
@@ -98,7 +98,10 @@ fs.writeFileSync(`${BUILD_DIR}/package.json`, JSON.stringify(packageJSON, null,
 
 // Add an extra package.json underneath ESM to indicate module type
 // This helps tools like Jest identify this code as ESM
+// (name and version are required for MFE)
```

Comment:
This version of auth-js fails Module Federation builds (via webpack 5). Seems related to https://github.com/webpack/webpack/issues/13457. Adding `name` and `version` to this "dummy" `package.json` fixes the issue